legacy-packages/title-block/package.json

```diff
@@ -44,6 +44,6 @@
   },
   "peerDependencies": {
     "@kaizen/design-tokens": "^2.10.3 || ^3.0.0 || ^5.0.0 || ^6.0.0 || ^7.0.0 || ^8.0.0",
-    "react": "^16.9.0"
+    "react": "^18.1.0"
```

Comment:
Do we want this to allow React 16.x, 17.x, or 18.x?

Reply:
I'm hoping to delete this component pretty soon. The only reason it was kept around was because we didn't have an Elm version of `draft-title-block-zen`, but now that Elm is gone from this repo I think we might be able to safely delete this.